config/config.go

```diff
@@ -319,44 +319,40 @@ type Node struct {
 		// Rollup is the address of the Hermez.sol smart contract
 		Rollup ethCommon.Address `validate:"required" env:"HEZNODE_SMARTCONTRACTS_ROLLUP"`
 	} `validate:"required"`
-	// API specifies the configuration parameters of the API
-	API struct {
-		// Address where the API will listen if set
-		Address string `env:"HEZNODE_API_ADDRESS"`
-		// Explorer enables the Explorer API endpoints
-		Explorer bool `env:"HEZNODE_API_EXPLORER"`
-		// UpdateMetricsInterval is the interval between updates of the
-		// API metrics
-		UpdateMetricsInterval Duration `validate:"required" env:"HEZNODE_API_UPDATEMETRICSINTERVAL"`
-		// UpdateRecommendedFeeInterval is the interval between updates of the
-		// recommended fees
-		UpdateRecommendedFeeInterval Duration `validate:"required" env:"HEZNODE_API_UPDATERECOMMENDEDFEEINTERVAL"`
-		// Maximum concurrent connections allowed between API and SQL
-		MaxSQLConnections int `validate:"required,gte=1" env:"HEZNODE_API_MAXSQLCONNECTIONS"`
-		// SQLConnectionTimeout is the maximum amount of time that an API request
-		// can wait to establish a SQL connection
-		SQLConnectionTimeout Duration `env:"HEZNODE_API_SQLCONNECTIONTIMEOUT"`
-	} `validate:"required"`
+	API                  APIConfigParameters                  `validate:"required"`
 	RecommendedFeePolicy stateapiupdater.RecommendedFeePolicy `validate:"required"`
 	Debug                NodeDebug                            `validate:"required"`
 	Coordinator          Coordinator                          `validate:"-"`
 }
 
+// APIConfigParameters specifies the configuration parameters of the API
+type APIConfigParameters struct {
+	// Address where the API will listen if set
+	Address string `env:"HEZNODE_API_ADDRESS"`
+	// Explorer enables the Explorer API endpoints
+	Explorer bool `env:"HEZNODE_API_EXPLORER"`
+	// ReadTimeout is the maximum duration for reading the entire
+	// request, including the body.
+	Readtimeout Duration `env:"HEZNODE_API_READTIMEOUT"`
+	// WriteTimeout is the maximum duration before timing out
+	// writes of the response.
+	Writetimeout Duration `env:"HEZNODE_API_WRITETIMEOUT"`
```

Comment:
Please add these new params to the readme:
https://github.com/hermeznetwork/hermez-node/blob/develop/config/README.md